$NetBSD: patch-ab,v 1.3 2006/01/14 02:52:42 rillig Exp $

The second hunk avoids an endless loop when there is no
way to read from stdin.

--- build/fping.sh.orig	2005-01-23 09:19:02.000000000 +0100
+++ build/fping.sh	2006-01-14 03:47:22.000000000 +0100
@@ -2,7 +2,7 @@
 
 	echo "Checking for fping ..."
 
-	FPING=""
+#	FPING=""
 
 	for DIR in /bin /usr/bin /sbin /usr/sbin /usr/local/bin /usr/local/sbin /opt/bin /usr/pkg/bin
 	do
@@ -41,7 +41,7 @@
 			echo "like 'sudo' to run fping."
 			echo ""
 			echo "Hobbit needs the fping utility. What command should it use to run fping ?"
-			read FPING
+			read FPING || exit 1
 		fi
 	done
 
